build-scripts/newChange.ts

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as child_process from 'child_process'
+import * as fs from 'fs-extra'
+import { join } from 'path'
+// tslint:disable-next-line:no-implicit-dependencies
+import readlineSync = require('readline-sync')
+import { v4 as uuid } from 'uuid'
+
+const directory = '.changes/next-release'
+
+enum ChangeType {
+    Test = 'test',
+    BreakingChange = 'Breaking Change',
+    Feature = 'Feature',
+    Bugfix = 'Bug Fix',
+    Deprecation = 'Deprecation',
+    Removal = 'Removal'
+}
+
+interface NewChange {
+    type: string,
+    description: string
+}
+
+function promptForType(): string {
+    const message = `
```

Comment:
Should we include a 'Cancel' option? Or does `readlineSync.question` give us that for free?

Reply:
Using `keyInSelect` instead of `question` would provide a few benefits:
* cancel
* make the response handling less magic-numbery
* tie the response directly to an input
* handle output formatting

Reply:
keyInSelect works